docs/api-gateway-v2/08-01-exposesecure.md

```diff
@@ -128,9 +128,9 @@ Follow the instructions in the tabs to deploy an instance of the HttpBin service
   spec:
     gateway: kyma-gateway.kyma-system.svc.cluster.local
     service:
-      name: httpbin-proxy
+      name: httpbin
       port: 8000
-      host: httpbin-proxy.kyma.local
+      host: httpbin.kyma.local
```

Comment:
You assume that the domain is `kyma.local`, but it can be any other domain. You already have an env `DOMAIN` storing this value.